qaes/encryption.py

```diff
@@ -16,10 +16,10 @@ def __init__(self, key):
         self.key = hashlib.sha256(key.encode()).digest()
         self.cipher = lambda iv: Cipher(algorithms.AES(self.key), modes.CBC(iv), backend)
 
-    def encrypt(self, plain: str) -> str:
+    def encrypt(self, plain: str, randomly=True) -> str:
         plain = plain.encode('utf-8')
         padded_raw = self._pad(plain)
-        iv = self.generate_iv()
+        iv = self.generate_iv() if randomly else self.constant_iv(plain)
```

Comment:
The encryption shouldn't know anything about the process of generating IV. It would be better to pass the argument to `generate_iv()` itself. This method should decide whether to generate a constant or random IV.

Reply:
Fixed.